### DISCUSSION TITLE
RTL page

### DISCUSSION THREAD

**Discussion by QusaiImayyah** [QuestPDF/QuestPDF#403] [Q&A] [answered]
How to RTL The All Page not content ?
Or 
FlowDirection RightToLeft ?

**Comment by MarcinZiabek** [comment, marked as answer]
You can use page-global settings: https://www.questpdf.com/api-reference/page#global-content-direction-rtl

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
